.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -2,9 +2,8 @@
 name: Feature request
 about: Suggest an idea for this project
 title: "[Feature Request]"
-labels: ''
-assignees: ''
-
+labels: ""
```

Comment:
Avoid making unnecessary changes.

Reply:
this change is being automatically added by vs code i tried to resolve it but vs code on ssving make it double quotes

Reply:
i have resolved it